go/processor/floria/processor.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package floria
+
+import (
+	"fmt"
+
+	"github.com/Fantom-foundation/Tosca/go/tosca"
+	"github.com/holiman/uint256"
+)
+
+func init() {
+	tosca.RegisterProcessorFactory("floria", newProcessor)
+}
+
+func newProcessor(interpreter tosca.Interpreter) tosca.Processor {
+	if interpreter == nil {
+		panic("interpreter is required")
```

Comment:
mmmm I think that if you run the program and see it crashing and the panic is just "interpreter is requires" seems a little out of context. Perhaps make a more self contained panic message, such as "interpreter is required to start a new processor" or something else. wdyt ?